addons/account/static/tests/reconciliation_tests.js

```diff
@@ -1793,92 +1822,5 @@ QUnit.module('account', {
 
         clientAction.destroy();
     });
-
-    QUnit.test('Manual Reconciliation: No lines for account', function (assert) {
```

Comment:
I don't understand why you had to remove this test? Adding the load more should not change the behavior of the create form being shown when no line is fetched.